src/main/drivers/pwm_rx.c

```diff
@@ -150,76 +174,76 @@ static void ppmEdgeCallback(timerCCHandlerRec_t* cbRec, captureCompare_t capture
     int32_t i;
 
     /* Shift the last measurement out */
-    ppmDev.previousTime = ppmDev.currentTime;
+    device.ppm.previousTime = device.ppm.currentTime;
 
     /* Grab the new count */
-    ppmDev.currentTime  = capture;
+    device.ppm.currentTime  = capture;
 
     /* Convert to 32-bit timer result */
-    ppmDev.currentTime += ppmDev.largeCounter;
+    device.ppm.currentTime += device.ppm.largeCounter;
 
     // Divide by 8 if Oneshot125 is active and this is a CC3D board
-    ppmDev.currentTime = ppmDev.currentTime >> ppmCountShift;
+    device.ppm.currentTime = device.ppm.currentTime >> ppmCountShift;
 
     /* Capture computation */
-    ppmDev.deltaTime    = ppmDev.currentTime - ppmDev.previousTime;
+    device.ppm.deltaTime    = device.ppm.currentTime - device.ppm.previousTime;
 
-    ppmDev.previousTime = ppmDev.currentTime;
+    device.ppm.previousTime = device.ppm.currentTime;
 
 #if 0
     static uint32_t deltaTimes[20];
     static uint8_t deltaIndex = 0;
 
     deltaIndex = (deltaIndex + 1) % 20;
-    deltaTimes[deltaIndex] = ppmDev.deltaTime;
+    deltaTimes[deltaIndex] = ppm.deltaTime;
```

Comment:
Missed while refactoring (inactive code fragment): `ppm.deltaTime` must be `device.ppm.deltaTime`